poly/src/domain/mixed_radix.rs

```diff
@@ -365,7 +325,7 @@ pub(crate) fn serial_mixed_radix_fft<T: DomainCoeff<F>, F: FftField>(
 
         // Doing the q_adicity passes.
         for _ in 0..q_adicity {
-            let w_m = omega.pow(&[(n / (q * m)) as u64]);
+            let w_m = omega.pow([(n / (q * m)) as u64]);
```

Comment:
As a future microoptimization, we can precompute `w_m` as a table outside the `for` loop, and save some multiplications. (Not clear if this would be worth it at all)